src/AWS.Deploy.Common/Recipes/OptionSettingItem.cs

```diff
@@ -160,13 +160,16 @@ public partial class OptionSettingItem : IOptionSettingItem
         /// Indicates which option setting items need to be validated if a value update occurs.
         /// </summary>
         public List<string> Dependents { get; set; } = new List<string> ();
+        
+        public  OptionSettingValidation Validation { get; set; }
```

Comment:
Given this class is already in pretty good shape - could you document this?

Reply:
Done